### DISCUSSION TITLE
Compatiability for Folder Overview with Custom File Explorer sorting

### DISCUSSION THREAD

**Discussion by TomBombTom** [LostPaul/obsidian-folder-notes#178] [Ideas]
Would it be possible to have the Folder overview read the custom sorting file from the [Custom File Explorer sorting](https://github.com/SebastianMC/obsidian-custom-sort) plugin? This would be a great use, as I have many folders which use this plugin, and also this one. However, the files/folders are only sortable by Name, Date Created, and Date Modified.

**Comment by ivxvm** [comment]
Yes please, `Custom File Explorer sorting` is the most popular sorting plugin, it has 200k downloads at this point. Would be really nice to have an option for folder overview to respect `sortspec` from this plugin.
